src/crystal/system.cr

```diff
@@ -8,10 +8,17 @@ module Crystal::System
   # def self.cpu_count
 end
 
-require "./system/unix/hostname"
+{% if flag?(:unix) %}
+  require "./system/unix/hostname"
 
-{% if flag?(:freebsd) || flag?(:openbsd) %}
-  require "./system/unix/sysctl_cpucount"
-{% elsif flag?(:unix) %}
-  require "./system/unix/sysconf_cpucount"
+  {% if flag?(:freebsd) || flag?(:openbsd) %}
+     require "./system/unix/sysctl_cpucount"
```

Comment:
Hmm there seems to be a little GitHub indentation bug?
I suggested 2 spaces (https://github.com/crystal-lang/crystal/pull/6972#discussion_r240786893) but now there are 3 here.
Can one space be removed? That'd be the last thing from me.

Reply:
Fixed in 2c59c80.